src/components/Navbar/Navbar.jsx

```diff
@@ -12,18 +12,19 @@ const  Navbar = () => {
                <Link to="/" class="nav-brand-name head">BuyWatch</Link>
                </div>
                <div class="nav-search-box ">
-               <input className="navbar-search" 
-               placeholder="Search for Products,Brands.."
-               />
-               <span class="material-icons search-icon">search</span>       
+                <p>Home</p>
+                <p>Products</p>
+                <p>Wishlist</p>
+                      
                </div>
                <div class="nav-right-head">
+               <span class="material-icons search-icon">search</span>
```

Comment:
I see below you're using react-icons are some other library here. Would recommend using one only, that'll unnecessarily increase the load on your website